common/common-lang/src/main/java/com/twelvemonkeys/util/TimeFormat.java

```diff
@@ -404,7 +404,7 @@ String format(Time t) {
 
         // Else return it with leading 0's
 	//return StringUtil.formatNumber(t.getSeconds(), digits);
-        return StringUtil.pad("" + t.getSeconds(), digits, "0", true);
+        return StringUtil.pad(String.valueOf(t.getSeconds()), digits, "0", true);
```

Comment:
Actual change. Using `String.valueOf()` instead of concatenating an empty string.

The code analysis states that For the purpose of turning something into a String, concatenating an empty string is never going to be as good performing as `String.valueOf()`

Reply:
`Integer.toString` might be even faster in this case, skipping one indirection... But I guess this would still be an improvement. :-)